examples/cli-auth-demo/src/main/java/com/databricks/example/CliAuthWorkspace.java

```diff
@@ -30,9 +30,9 @@ private static DatabricksConfig getConfig() {
      Authenticate and retrieve the list of clusters from the workspace
      */
     public static void main(String[] args) {
-        DatabricksConfig config = getConfig();
+        // DatabricksConfig config = getConfig();
```

Comment:
Should we remove this comment from demo?

Reply:
Right, thanks for the reminder, I was going to undo this change